src/components/sections/BannerText/banner-text.scss

```diff
@@ -0,0 +1,47 @@
+@import "../../../styles/vendors/include-media.scss";
+
+[data-store-banner] {
```

Comment:
Not sure! But I think we want to scope this only for the `BannerText`? If so we need to add `banner-text` class to  scope it.